client/src/pages/Membership/HowToJoin/Step.tsx

```diff
@@ -1,11 +1,8 @@
+import { /*type*/ WithStyles } from '@material-ui/core';
+import { /*type*/ ImgIcon } from 'types/imgIcon';
+
 import React from 'react';
-import {
-  Typography,
-  createStyles,
-  WithStyles,
-  withStyles,
-  Theme,
-} from '@material-ui/core';
+import { Typography, createStyles, withStyles, Theme } from '@material-ui/core';
```

Comment:
I feel like I'm adding plates to the sink while you're washing dishes but...

`Theme` is a type too😅🙈

Reply:
🤣

I'm not a fan of separating types until VSCode and TS automatically do it for me. 😢 

fixed!